handler.go

```diff
@@ -39,6 +40,7 @@ func (e EventHandler) Handle() http.Handler {
 func (e EventHandler) CheckToken(token string, next http.Handler) http.Handler {
 	return http.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
 		if token != request.URL.Query().Get("token") {
+      log.Printf("Wrong token : %s != %s", token, request.URL.Query().Get("token"))
```

Comment:
Better to not log token if its related to authorization and accesses

Reply:
but this is visible only for the one who is starting the server.
And this is not auth token from bitbucket, simple token to avoid calling this service by any one